account_account_constraint_code/tests/test_account_account.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022 Acsone SA (http://www.acsone.eu/)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from datetime import datetime
+
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestAccountAccount(TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.misc_journal = self.env["account.journal"].create(
```

Comment:
Can you add this: `self.env = self.env(context=dict(self.env.context, tracking_disable=True))`